todoapp/urls.py

```diff
@@ -1,5 +1,11 @@
-from django.urls import path,include
+from django.urls import path
+from . import views
 
 urlpatterns = [
     
+    path('get_todo/' , views .get_todo),
```

Comment:
Basically we have two view class

TodoListView include get and post method (get for listing the todo)
TodoDetailView include put,patch and delete method